.github/workflows/test-fork-prs.yml

```diff
@@ -95,18 +96,104 @@ jobs:
             return commit_id
               .replace(/(\r\n|\n|\r)/gm, '') // remove last new line character
               .replace(/[^A-Za-z0-9]/g, ''); // remove non-alphanumeric characters
-      # create status
+      - name: Read PR number from artifact
+        uses: actions/github-script@98814c53be79b1d30f795b907e553d8679345975
+        id: read-pr-number
+        with:
+          result-encoding: string
+          script: |
+            const fs = require('fs');
+            const pr_number = fs.readFileSync('./pr_number', 'utf-8');
+            return pr_number
+              .replace(/(\r\n|\n|\r)/gm, '') // remove last new line character
+              .replace(/[^0-9]/g, ''); // remove non-alphanumeric characters
+      - name: Read base sha from artifact
+        uses: actions/github-script@98814c53be79b1d30f795b907e553d8679345975
+        id: read-base-sha
+        with:
+          result-encoding: string
+          script: |
+            const fs = require('fs');
+            const base_sha = fs.readFileSync('./base_sha', 'utf-8');
+            return base_sha
+              .replace(/(\r\n|\n|\r)/gm, '') // remove last new line character
+              .replace(/[^A-Za-z0-9]/g, ''); // remove non-alphanumeric characters
+
       - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c #v3.3.0 https://github.com/actions/checkout/commit/ac593985615ec2ede58e132d2e21d2b1cbd6127c
       - name: Set status to commit sha
         uses: ./.github/actions/set-status
         with:
-          sha: ${{ steps.read-artifact.outputs.result }}
+          sha: ${{ steps.read-commit-id.outputs.result }}
           state: 'pending'
-          context: 'Run E2E tests'
-          description: 'E2E tests are now running'
+          context: 'Run PR checks'
```

Comment:
As we changed this file to run PR suite instead of just e2e, we changed the name to generic "Run PR checks". This will be required for PR to merge.